doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,248 @@
+| outband_subnet_mask | String | Yes |  |
+| pim_username | String | Yes |  |
+| pim_password | String | Yes |  |
+| rack_name | String | Yes |  |
+| position | String | Yes |  |
+| remote_manange_ip | String | Yes | remote management ip, e.g. iLO,iDRAC, BMC |
+| remote_user | String | Yes | remote user  |
+| remote_password | String | Yes | remote password |
+
+<p align="center"><b>Table 5-3-3:</b> Server information.</p>
+
+
+### 5.3.4 Software configuration definition
+It includes Virtualized Infrastructure Manager configurations,
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| vim_name | String | Yes | vim name |
+| vim_id | String | No | a name VIM Resource pool naming rule |
+| vendor | String | Yes | VIM Provider information, e.g. ZTE,Huawei,Ericsson,NOKIA|
+| version | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| ip_version | String | Yes | Ipaddress type: IPV6 or IPV4 |
+| az_info | List | Yes | list of availability zone name designed for this VIM |
+| ha_name | List | No | HA name  if provided, HA is set of host server group having one or more same attribute(s),e.g: hardware specification.|
+| network_infos | List | Yes | List of NICs used in VIM deployment |
+| controller_nodes | List | Yes | List of controllers designed in VIM deployment |
+| compute_nodes | List | Yes | List of compute nodes designed in VIM deployment |
+
+<p align="center"><b>Table 5-3-4:</b> Software configuration.</p>
+
+
+### 5.3.5 Network informations
+List of All designed NIC definitions which are referenced by various roles of node, control/compute/network/storage node.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| network_plane_type | String | Yes | distinguish network type: e.g MANAGEMENT,STORAGEDATA |
+| usage | String | Yes | used for compute/control: value is one of [manage,storage,service], for distribute storage  value if one of [storage_front,storage_backend] |
+| network_plane_infos | List | Yes | vlan_id,sub_network(CIDR),gateway(could be null)|
+
+<p align="center"><b>Table 5-3-5-1:</b> Network information.</p>
+
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| vlan_id | String | Yes | vlan id for current network |
+| sub_network | String | Yes | subnet in CIDR format.|
+| gateway | String | No | gateway(could be null)|
```

Comment:
Can gateway be null ?

Reply:
to check further, I will remove null from it at present.